examples/text-generation/README.md

```diff
@@ -155,7 +155,8 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 --use_hpu_graphs \
 --use_kv_cache \
 --batch_size 1 \
---do_sample
+--do_sample \
+--use_flash_attention --flash_attention_causal_mask
```

Comment:
please add these to run_lm_eval.py, too

Reply:
@schoi-habana the fusedsdpa fp8 is not ready for falcon yet